go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/devfile/devworkspace-operator
 
-go 1.22
-
-toolchain go1.22.0
```

Comment:
Why is the toolchain version omitted?

Reply:
I think this is a good point. Could you add the `toolchain` directive here so that we can merge this PR?

Reply:
@rohanKanojia I noticed this line in the [golang toolchain doc](https://tip.golang.org/doc/toolchain#get):

> When the toolchain matches the `go` line exactly, it can be omitted and implied, so this `go get` will delete the `toolchain` line.

I also checked the `go mod tidy` behaviour, and it works the same way. Hence, the pull request check will fail if we add the `toolchain` directive with the same version:

https://github.com/devfile/devworkspace-operator/blob/9ff4cee22e67cb71d7ed6ba6e473308fbcd8ae8e/.github/workflows/pr.yml#L62-L71

So, we can merge this PR as is.

Reply:
Okay, thanks for the clarification!